index.js

```diff
@@ -51,16 +51,24 @@ const body = {
   parameters: parameters,
 };
 
+const tag = getTag();
+const branch = getBranch();
+
 if (tag) {
```

Comment:
Previously this was always truthy as we were referencing the function rather than its return value.